migrations/20191031154616-CreateEtityTable.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function(db) {
+  return db.createTable('entity', {
```

Comment:
1. We don't need person_id or platner_organization_id since these both collapse into entity_id due to our single table (2 view) implementation of entities
2. 'entity_id' and 'id' in this table would be synonyms.. So we don't need entity_id

Reply:
1. Ahh, yes. I was worried about the connection to the planter (user) table but if the number generated from the entity table are unique then that makes sense. How will we distinguish/filter the two table apart in the query to form the views. My thoughts were having a type column (type: o/p) to separate person from organization (planting_organization).    

2: Oops, sorry about that. Noted and will change. Got carried away with the entity_id.

Reply:
A type column is the correct approach, we do need to add one.